QUICK_START.md

```
@@ -0,0 +1,137 @@
+The test page demonstrates:
+- ✅ How `showErrors()` highlights form fields
+- ✅ How `ajaxPost()` handles success and failure
+- ✅ Side-by-side comparison of old vs new pattern
+- ✅ Benefits of the new approach
+
+### Step 3: Use in Your Code
+
+Copy this basic pattern:
+
+```javascript
+$('#btn_submit').click(function () {
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)